entities/room.entities.js

```diff
@@ -5,7 +5,7 @@ const { Schema, model } = mongoose;
 const RoomSchema = new Schema({
   name: { type: String },
   enabled: { type: Boolean },
-  maxCapacity: { type: Number },
+  maxCapacity: { type: Number, default: 10 },
 });
```

Comment:
I see that officeschema's default is `default: true`, should this schema have the same?